packages/slice-machine/lib/forms/fields.ts

```diff
@@ -51,7 +51,7 @@ export interface InputType {
   label: string;
   validate: any;
   defaultValue: string | undefined;
-  fieldLevelValidation: FieldLevelValidationFn | undefined;
+  fieldLevelValidation: FieldLevelValidationFn | undefined | null;
```

Comment:
what's the reason for this? 🤔

Reply:
`null` was being passed as a value for fieldLevelValidation in a form. Didn't wanna change it to `undefined ` cuz I am not too sure if it could've caused troubles